content/environment_variables.tex

```diff
@@ -31,13 +31,25 @@
 %%
 \EnvVarDecl{SHMEM\_SYMMETRIC\_SIZE}
     & Non-negative integer
-    & Number of bytes to allocate for symmetric heap
+    & Controls the size of the symmetric heap per PE. The value
+      set in this environment variable is interpreted as number
+      of bytes, unless the number is followed by a character that
```

Comment:
Suggest: "a **single** character"

Reply:
marking as resolved - as the text has gone multiple updates and this one is no longer relevant.